.prettierrc

```diff
@@ -1,9 +1,11 @@
 {
-  "printWidth": 100,
-  "semi": false,
+  "printWidth": 80,
+  "semi": true,
   "singleQuote": true,
-  "bracketSpacing": false,
```

Comment:
makes things more readable imo

Reply:
Ha, this one is funny. I looked it up in the docs, because `bracketSpacing` sounds as if it would add spaces to brackets, which I prefer not to have, but it does seem to add them to braces, which is nice.